src/lib.rs

```diff
@@ -315,59 +321,90 @@ pub fn run(config: Config) -> Result<(), Box<dyn Error>> {
     Ok(())
 }
 
-#[derive(Debug, PartialEq)]
-enum CliCommand {
-    Start,
-}
+pub fn parse_args<I, T>(args: I) -> CliArgs
+where
+    I: IntoIterator<Item = T>,
+    T: Into<OsString> + Clone,
+{
+    let app_matches = App::new(clap::crate_name!())
+        .version(clap::crate_version!())
+        .author(clap::crate_authors!())
+        .about(clap::crate_description!())
+        .setting(AppSettings::SubcommandRequiredElseHelp)
+        .subcommand(
+            SubCommand::with_name("start")
+                .about("Start a new tmux session")
+                .arg(
+                    Arg::with_name("PROJECT_FILE")
+                        .help("The session project file to launch")
```

Comment:
This applies to a bunch of different parts of the project, but it'd be good to standardize a name for this concept. I've been using "project config file" here (README, the Config struct, etc.) and in tmuxinator, but that also doesn't mean it's the best choice. The file is being used to do a number of similar but related things (inside this project and in the context of tmux), so that does make giving it a clear, simple name difficult.

Reply:
I can update it to _project config file_ unless you have a better alternative that you want to use. The meaning being obvious from the name is all that really matters in the end.